### PR TITLE
VOTE-695: Update Drupal core, Drush, and contrib modules.

### PR DESCRIPTION
**Pull request by daniel-crowder**
## Jira ticket

[VOTE-695](https://cm-jira.usa.gov/browse/VOTE-695)

## Description

Upgrade Drupal core and contrib modules to current versions.

### Pre-deploy

1. Insert any required steps to take before merging and running post-deploy.

### Post-deploy

1. Locally, run`lando retune -y`

### QA/Test

1. Browse site locally to confirm that everything works as expected.
2. Login as an administrator, view Available Updates page (/admin/reports/updates) and confirm that modules and core are updated to latest versions.
3. Review Status Report page (/admin/reports/status) and confirm no significant errors or warnings are listed.

1. On cloud environments, confirm that media file uploads work and are accessible via S3 path when clicking on the link for the file from the media files admin list page.

## Checklist for the Developer

- [x] A link to the JIRA ticket has been included above.
- [x] No merge conflicts exist with the target branch.
- [x] Automated tests have passed on this PR.
- [x] A reviewer has been designated.
- [x] Deployment and testing steps have been documented above, if applicable.

## Checklist for the Peer Reviewers

- [x] The file changes are relevant to the task objective.
- [x] Code is readable and includes appropriate commenting.
- [x] Code standards and best practices are followed.
- [x] QA/Test steps were successfully completed, if applicable.
- [x] Applicable logs are free of errors.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -57,7 +57,7 @@
         "drupal/focal_point": "^2.0",
         "drupal/language_switcher_extended": "^1.1",
         "drupal/link_attributes": "^2.1",
-        "drupal/log_stdout": "dev-3339853-drupal-10-compatibility",
+        "drupal/log_stdout": "dev-8.x-1.x",
```
Comment (rayestrada) [inline composer.json:60]: Can you check if D10 compatibility is in the stable version 8.x-1.5?
It seems to be suggested here https://www.drupal.org/node/3339853

File: composer.json
Comment (rayestrada) [inline composer.json:1]: Looks like publication_date module patch was merged and released
https://git.drupalcode.org/project/publication_date/-/merge_requests/12

Comment (rayestrada) [inline composer.json:1]: Looks like pathauto patch was merged and released
https://www.drupal.org/node/3328670

**Review by rayestrada — CHANGES_REQUESTED** [review]
@daniel-crowder can you check if we can upgrade these modules to stable releases?

File: composer.json (hunk 1 above)
Comment (daniel-crowder) [reply composer.json]: Updated!

File: composer.json
Comment (daniel-crowder) [reply composer.json:1]: Updated!

Comment (daniel-crowder) [reply composer.json:1]: Updated!

**Review by rayestrada — DISMISSED** [review]

**Review by rayestrada — APPROVED** [review]

**Pull request merged**